### PR TITLE
Fix subpatch alignment for delay bangs

### PR DESCRIPTION
**Pull request by AlexMitchellMus**

### CODE REVIEW

**Comment by tomara-x** [comment]
hmm it would seem the open on load flag is set to 1 on all those graphs.. so if you try opening this in vanilla, you get a window explosion

it seems plugdata isn't updating it. so i can't fix it by loading the patch, only having the main window open, and then saving it. (a bug to investigate on your end)

anyways, this is a place of wabi-sabi so who cares (there are much bigger issues with that patch and this repo in general, i wouldn't waste any time fixing things in it)

File: things/girls/amaranth/amaranth.pd [hunk 1]
```diff
@@ -1012,16 +1012,16 @@
 #X restore 358 446 pd record;
 #X obj 403 416 tgl 25 0 empty empty empty 17 7 0 10 #191919 #ffffff #ffffff 0 1;
 #X symbolatom 358 472 10 0 0 0 - - - 0;
-#N canvas 0 50 450 250 (subpatch) 0;
+#N canvas 0 50 450 250 (subpatch) 1;
```
Comment (tomara-x) [inline things/girls/amaranth/amaranth.pd:1015]: that's an open-on-load flag

Comment (AlexMitchellMus) [reply things/girls/amaranth/amaranth.pd:1015]: How did you make this patch with open-on-load false? Did you edit it in PD-Vanilla?

Comment (tomara-x) [reply things/girls/amaranth/amaranth.pd:1015]: no, it used to work in plugdata, so this is a regression

**Pull request merged**